testing/test_dataset.py

```diff
@@ -55,6 +54,100 @@ def _load_file(filePath):
     return out
 
 
+def flipGains(doc: Dataset, idChanges: tuple):
+    """ Utility method to flip the gains of requested calibration IDs """
+
+    for id in idChanges:
+        currentCoeffs = doc.transforms[id].coefficients
+        # flipping only the gain - univariate or bivariate
+        if len(currentCoeffs) == 2:
+            adjustedCoeffs = (-currentCoeffs[0], currentCoeffs[1])
+        else:
+            adjustedCoeffs = (currentCoeffs[0], -currentCoeffs[1], currentCoeffs[2], currentCoeffs[3])
+        doc.transforms[id].coefficients = adjustedCoeffs
+    doc.updateTransforms()
+    return doc
+
+
+@pytest.fixture(scope="session")
```

Comment:
last commit uses pytest fixtures; second to last uses global variables & takes some ms longer to run.
there's a comment explaining these two processes under https://midetech.atlassian.net/browse/ES-548.

Reply:
Do you mean global _variables_, or module-level _functions_ (as opposed to methods of a class)?  I'd say global variables are also acceptable (in moderation) in cases like this, so don't worry too much. The functions are _definitely_ fine; the `unittest.TestCase` subclasses are basically legacy, and most new pytest tests just consist of module-level definitions of things.

Regarding speed: If the differences in run time aren't increased geometrically, don't worry too much about things being a bit slower.

Reply:
Sounds great thank you! 
I meant global variables. In the commit prior to the "switching to pytest fixtures," I had set up too many massive global dicts; two were storing the MMM arrays for all channels of either the Xformed IDE or the original IDE, two dicts to hold the expected MMMs for all channels of both of these files, & then another dict to hold the expected shape of each channel's MMM array. 
I wanted to use fixtures from the start, but was having some initial trouble since I couldn't parametrize fixtures